tests/integration/test_schema.py

```diff
@@ -2316,21 +2322,26 @@ async def test_invalid_schema_should_provide_good_error_messages(registry_async_
     )
     assert res.json()["message"] == "Invalid AVRO schema. Error: Expecting ',' delimiter: line 1 column 18 (char 17)"
 
-    # Unfortunately the AVRO library doesn't provide a good error message, it just raises an TypeError
     schema_str = jsonlib.dumps({"type": "enum", "name": "error"})
     res = await registry_async_client.post(
         f"subjects/{test_subject}/versions",
         json={"schema": schema_str},
     )
-    assert res.json()["message"] == "Invalid AVRO schema. Error: Provided schema is not valid"
+    assert (
+        res.json()["message"]
+        == "Invalid AVRO schema. Error: Enum symbols must be a sequence of strings, but it is <class 'NoneType'>"
```

Comment:
This is not understandable error message if received by Schema Registry users

Reply:
This looks like this is something that should be improved upstream in the Avro Python SDK.  I can raise a JIRA, but I would suggest either (1) rewriting all Avro error messages systematically in karapace if we want them to be consistent, or (2) passing them on as-is if we want them to be "searchable" in a search engine.

There's currently a proposal to improve and unify some error messages across the different language SDKs, but it wouldn't be ready in the next minor release.

Reply:
+1 for improving this upstream. The message is not drastically worse and it is probably fine to wait for the next release.

Reply:
Just for info -- [this is the proposition](https://github.com/apache/avro/pull/1526#issuecomment-1039668331) for some error messages for the Java SDK.  I'm hoping that the syntax is adopted across language SDKs.

Reply:
I am for the upstream change just because of the option 2 has the searchability.

Reply:
Ok, we can go with this, and wait for the upstream improving the error messages. At least here the error message is also improvement over old one unlike with reserved keyword (`"error"` being reserved word) case later.